src/routes/captures.router.ts

```diff
@@ -12,7 +12,7 @@ const captureRouter = Router();
 
 // Create new flamegraph from a .perf
 captureRouter.post(
-  '/flamegraph',
+  '/:flamegraph',
```

Comment:
Did you intentionally add the colons (here and for /api/capture/icicle) back? I think they might cause a bug with the way req.params is handled